internal/commands/secrets/create_inputs_test.go

```diff
@@ -0,0 +1,102 @@
+package secrets
+
+import (
+	"testing"
+
+	"github.com/10gen/realm-cli/internal/cli"
+	"github.com/10gen/realm-cli/internal/utils/test/assert"
+	"github.com/10gen/realm-cli/internal/utils/test/mock"
+	"github.com/Netflix/go-expect"
+)
+
+func TestCreateInputs(t *testing.T) {
+	for _, tc := range []struct {
+		description    string
+		createInputs   createInputs
+		prepareProfile func(p *cli.Profile)
+		procedure      func(c *expect.Console)
+		test           func(t *testing.T, i createInputs)
+	}{
+		{
+			description: "should prompt for secret name when not provided",
+			createInputs: createInputs{
+				Value: "value",
+			},
+			prepareProfile: func(p *cli.Profile) {},
+			procedure: func(c *expect.Console) {
+				c.ExpectString("Secret Name")
+				c.SendLine("name")
+			},
+			test: func(t *testing.T, i createInputs) {
+				assert.Equal(t, "name", i.Name)
+			},
+		},
+		{
+			description: "Should prompt for secret value when not provided",
+			createInputs: createInputs{
+				Name: "name",
+			},
+			prepareProfile: func(p *cli.Profile) {},
+			procedure: func(c *expect.Console) {
+				c.ExpectString("Secret Value")
+				c.SendLine("value")
+				c.ExpectEOF()
+			},
+			test: func(t *testing.T, i createInputs) {
+				assert.Equal(t, "value", i.Value)
+			},
+		},
+		{
+			description:    "Should prompt for both secret name and secret value when not provided",
+			prepareProfile: func(p *cli.Profile) {},
+			procedure: func(c *expect.Console) {
+				c.ExpectString("Secret Name")
+				c.SendLine("name")
+				c.ExpectString("Secret Value")
+				c.SendLine("value")
+				c.ExpectEOF()
+			},
+			test: func(t *testing.T, i createInputs) {
+				assert.Equal(t, "name", i.Name)
```

Comment:
[nit] could make this `assert.Equal(t, createInputs{"name",  "value"}, i)`  (and could likewise do the same for the above tests... just gives a little more assurance we aren't modifying more things than we expect